orm/models.py

```diff
@@ -190,6 +197,10 @@ def select_related(self, related):
             raise MultipleMatches()
         return self.model_cls.from_row(rows[0], select_related=self._select_related)
 
+    async def order_by(self, *args, **kwargs):
+        self._order_args = args
+        return await self.all(**kwargs)
```

Comment:
We shouldn't have `order_by` run `.all(...)`, so let's just do:

```python
def order_by(self, *args):
        self._order_args = args
```

Reply:
Actually we should include `order_args` in the `__init__`, and create a new queryset in the same way we're currently doing in `filter` and `select_related`